.ruby-version

```diff
@@ -0,0 +1 @@
+3.1.3
```

Comment:
What's this then?

Reply:
Sorry, was hoping on the commit to document why this was here. We didn't have anything in the team docs to set which version of Ruby to run against, like in our Frontend Docs, meaning if you had an old version by default, things wouldn't run locally. 

I could split that in a separate PR if we prefer 😊

Reply:
Ah gotcha! I think that's probably fine to pop in here since it's in it's own commit.